### PR TITLE
fix responsiveness issue #22

### LINKED ISSUE #22: Issue : Responsiveness issue on certain Device also Nav items taking whole space
I have reviewed your portfolio. Though I found some issues which I can fix.

Issues : contents not managed on certain device, also your navbar is bulky for mobile devices.

Please assign this issue to me and tag it with hacktoberfest !! 

Looking forward to contribute in your project

### PR DESCRIPTION
**Pull request by c4coderandcreator**
## Pull Request

**Summary:**
updated the nav Items on smaller screen for better user experience , fixed the para alignment, fixed the overlapping items at certain screen sizes.

**Related Issue:**
Fixes #22 

**Changes Made:**
- fixed nav items
- para alignment
- overlapping items

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -121,6 +121,7 @@ nav.scrolled {
 .about-col-1 img {
   width: 100%;
   border-radius: 15px;
```
Comment (soniyaprasad77) [inline static/css/styles.css:123]: I believe the image is more effective in its current location. I recommend retaining the original code for its placement.

Comment (c4coderandcreator) [reply static/css/styles.css:123]: @soniyaprasad77  but your right section is ahead of it ( vertical alignment doesn't match ) !! That's why I tried to match it !! Though if you don't want anything with images i cant revert back those changes.

Comment (c4coderandcreator) [reply static/css/styles.css:123]: @soniyaprasad77  Extremely sorry that missed to remove that border-radius thing !! I was just testing out the round image on your portfolio ! It is fixed now !! Thanks for pointing out

**Comment by soniyaprasad77** [comment]
nav


**I believe the vertical navbar configuration I've suggested in the above picture is the optimal layout. Additionally, it's advisable to eliminate any margin on the right-hand side of the navbar, as indicated by the small pink dashes. Please implement these adjustments.**

**Comment by c4coderandcreator** [comment]
The Issue you pointed out is Browser compatibility issue , basically its not an code issue you just need to pinch in and out and it will vanish !! It basically appears when users are checking responsiveness , see the image attached without any changes those gaps are not there 
image

**Comment by soniyaprasad77** [comment]
Alright... I will check, just make the other changes. 😊

**Comment by c4coderandcreator** [comment]
Just confirming !! You want this ? 
image

**Comment by soniyaprasad77** [comment]
Yeah

**Comment by c4coderandcreator** [comment]
@soniyaprasad77  It is Updated !! Please check and merge !!

**Comment by c4coderandcreator** [comment]
It was my pleasure working with you @soniyaprasad77

**Pull request merged**